test/e2e/pages/cancel-move.js

```diff
@@ -39,7 +39,10 @@ class CancelMovePage extends Page {
           selector: this.fields.cancellationReasonOtherComment,
           value: comment,
         }
-      } else if (reason === 'Cancelled by Population Management Unit (PMU)') {
+      } else if (
+        reason === 'Cancelled by Population Management Unit (PMU)' &&
```

Comment:
Where on earth does this reason come from?

Reply:
aaaah. It's the e2e tests. Move along... nothing to see here

Reply:
Yeah, they work off labels, so changing the `locale` files will break the E2E tests, which is nice.